### PR TITLE
Potential Notice error

### PR DESCRIPTION
**Pull request by voxTeam**

### CODE REVIEW

File: shoppingfluxexport.php [hunk 1]
```diff
@@ -498,11 +498,14 @@ private function countProducts()
 			else
 				$front = true;
 
+			$sql_association = Shop::addSqlAssociation('product', 'p');
+			$table = $sql_association ? 'product'.'_shop' : 'p';
```
Comment (Quetzacoalt91) [inline shoppingfluxexport.php]: You do not need to concatenate 'product' and '_shop'. Write them together directly: `product_shop`

**Comment by Quetzacoalt91** [comment]
Hi @clement-hollander,

Because you usually post new releases of this module, I let you check and test this pull-request before merging anything.

Best regards

**Comment by clement-hollander** [comment]
Hi @Quetzacoalt91 , I agree these changes

**Comment by Quetzacoalt91** [comment]
Hi @voxTeam and @clement-hollander,

Thanks, I will now merge the pull-request and the changes will be available from tomorrow.

Best regards

**Pull request merged**